Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -8,18 +8,28 @@
 
 #include <AzToolsFramework/API/PythonLoader.h>
 #include <AzCore/Debug/Trace.h>
+#include <AzFramework/StringFunc/StringFunc.h>
```

Comment:
Actually prefer to use StringFunc from AzCore and not AzFramework.

Or better yet, use AZ::IO::PathView